apps/settings/js/panels/bluetooth/bt_template_factory.js

```diff
@@ -18,7 +18,7 @@ define(function() {
   function btTemplate(deviceType, onItemClick, observableItem) {
     var device = observableItem;
 
-    var nameSpan = document.createElement('span');
+    var nameSpan = document.createElement('bdi');
```

Comment:
bt_template_factory is used for both "Paired devices" and "Devices in the Area" section, so we just need patch once